components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/util/IdpSearchResult.java

```diff
@@ -0,0 +1,75 @@
+package org.wso2.carbon.idp.mgt.util;
+
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+
+import java.util.List;
+
+/**
+ * Pagination response.
+ */
+public class IdpSearchResult {
```

Comment:
Since this is a POJO class, shall we move it to a different sub folder, than util folder.?